app/vue/README.md

```diff
@@ -28,7 +28,13 @@ You can also build a [static version](https://storybook.js.org/docs/vue/workflow
 
 ## Vue Notes
 
-- When using global custom components or extensions (e.g., `Vue.use`). You will need to declare those in the `./storybook/preview.js`.
+When using global custom components or extensions, you will need to declare those in the `./storybook/preview.js` like so:
+
+```js
+import { app } from "@storybook/vue3";
+import VueExtensionOrComponent from ...;
+app.use(VueExtensionOrComponent);
+```
```

Comment:
@Burtchen it seems that with the proposed change we're inbound to introduce a bit of a misconception. To give you a bit of context. This folder in the repo is related to Vue 2. We have something related in the Vue 3 folder, more specifically [here](https://github.com/storybookjs/storybook/tree/next/app/vue3). If you could adjust the snippet accordingly to Vue 2. I'm more than ok to merge this.

Let me know. 

Stay safe

Reply:
Oh. This is at least the third time (not for this PR, but in general) Vue2/3 confusion in docs bites me. In this case, through no fault of yours of course. 

That being said, since my work is only about a vue 3 extension in conjunction with storybook, I have nothing to contribute here actually. This can be closed.

Thanks for your swift and polite response, and for storybook!